<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fileDialog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>601</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>fileDialog</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepXLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Шаг по X</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepYLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Шаг по Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="paintLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="stepXSpinBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="stepYSpinBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Click</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Нормализация</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minNormalSpinBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="maxNormalSpinBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Чтобы вращать фигуру, используйте клавиши W, A, S, D&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Для масштабирования клавиши +-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="triglabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Градус поворота:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trigSpinBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>190</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="shiftXLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сдвиг по X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сдвиг по Y:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="shiftXSpinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="shiftYSpinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
